doc/nrf/app_dev/device_guides/fem/index.rst

```diff
@@ -7,26 +7,92 @@ Developing with Front-End Modules
 
 .. include:: /includes/guides_complementary_to_app_dev.txt
 
-Zephyr and the |NCS| provides support for developing applications with the following :term:`Front-End Module (FEM)` devices:
+Zephyr and the |NCS| provides support for developing applications with the following :term:`Front-End Module (FEM)` devices according to :ref:`software_maturity`:
 
-.. list-table::
-   :header-rows: 1
+.. tabs::
```

Comment:
Could be moved to the Software maturity page and then the contents could be pulled here using  ..include:: (to show it in both places).

Reply:
Discussed offline.